tests/python/gaia-ui-tests/gaiatest/apps/messages/app.py

```diff
@@ -11,6 +11,7 @@
 class Messages(Base):
 
     name = 'Messages'
+    manifest_url = "app://sms.gaiamobile.org/manifest.webapp"
```

Comment:
@mwargers is right. `self.DEFAULT_APP_HOSTNAME` exist in order to have one place to modify if we changed the hostname, one day. Not using it leads to duplication.